### ISSUE TITLE
Use xeCJK for fontenc as default with XeLaTex and Chinese docs

### ISSUE THREAD

**Issue by zhsj** [sphinx-doc/sphinx#6239]
When [building Chinese translation of CPython docs](https://github.com/python/docsbuild-scripts/pull/70). The only valid option I tried out is using xeCJK as the fontenc and using xelatex.

A simple project created by sphinx-quickstart, with some Chinese words(like `测试`) in `source/index.rst`, will fail the `make latexpdf` command.

Howerver the following command works,

```
make latexpdf SPHINXOPTS='-D latex_engine=xelatex -D latex_elements.fontenc=\\usepackage{xeCJK} -D latex_elements.inputenc='
```

Although I still get warning like `no Babel option known for language 'zh-cn'`, but the PDF looks good.

Could you set these as default for Chinese documents?

xeCJK will use Fandol font as default, both are Tex packages

https://ctan.org/pkg/xecjk
https://ctan.org/pkg/fandol

**Comment by zhsj** [comment]
Peviously the CPython docs are built with `-D latex_engine=xelatex -D latex_elements.fontenc= -D latex_elements.inputenc=`, the command will succeed, but the PDF doesn't contain any Chinese characters. I don't know much about latex, but I recall that I have used xeCJK before...

**Comment by tk0miya** [comment]
Thank you for the pointer. Okay, let's follow the python-doc's setting.

refs: #1119

**Comment by tk0miya** [comment]
Note: Sphinx expects the joint character of `language`  is `_` (underscore). So `zh_CN` is better for this case. But it does not resolve the warning.

**Comment by tk0miya** [comment]
Fixed by #6398.
Thank you for reporting!
